### PR TITLE
reenable classif label switch tests (closes #973)

### LINKED ISSUE #973: small unit test issues
after the new travis container build i did this (so we dont get blocked by this)

1) the classiflabelswitch test sometimes breaks with h2o.(deeplearning?) and dbnnet, at least. it is completely disabled now. I also removed the h2o learners from classiflabelswitch.

2) i have now set a seed in helpers_zzz 

both things should be reset. and investigated locally. 

the diff is here (for reference): 
https://github.com/mlr-org/mlr/pull/964/files

### PR DESCRIPTION
**Pull request by larskotthoff**

### CODE REVIEW

File: tests/testthat/test_learners_classiflabelswitch.R [hunk 1]
```diff
@@ -62,7 +62,9 @@ test_that("no labels are switched", {
       hps = hpars[[id]]
       if (!is.null(hps))
         lrn = setHyperPars(lrn, par.vals = hps)
-      err = holdout(lrn, task, split = 0.5, stratify = TRUE)$aggr[[1L]]
+      tmp = holdout(lrn, task, split = 0.5, stratify = TRUE)
```
Comment (jakob-r) [inline tests/testthat/test_learners_classiflabelswitch.R]: maybe revert this change

Comment (larskotthoff) [inline tests/testthat/test_learners_classiflabelswitch.R]: Well, it makes debugging easier because you just have to uncomment the one line below.

**Comment by berndbischl** [comment]
travis fails, but can be merged if greeen
